rpc/walletrpc/api.pb.go

```diff
@@ -652,6 +656,8 @@ var (
 		"StakeSubChangeTy":  9,
 		"PubkeyAltTy":       10,
 		"PubkeyHashAltTy":   11,
+		"TreasuryAddTy":     12,
+		"TreasuryGenTy":     13,
```

Comment:
I updated these in the DecodeTransaction ScriptClass enum as well, except I have these two switched in order.  Minor conflict to resolve, but wanted to point it out.

Reply:
Thank you! Will do this when/if  #2091 goes in.

Reply:
This should be resolved. Not related to the rpc side I believe.